<ui version="4.0" >
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PyKafe Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../data/icons/pyKafe.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QCheckBox" name="checkBox_2" >
         <property name="text" >
          <string>Ask password when opening</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" >
        <widget class="QCheckBox" name="checkBox" >
         <property name="text" >
          <string>Tray Icon</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Regional</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="2" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>21</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="lineEdit_4" />
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="lineEdit_3" />
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="lineEdit_2" />
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="lineEdit" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Decimals Seperator:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Thousands seperator:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Currency prefix:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Currency suffix:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Pricing</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Fixed price for first 30 minutes:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Price for one hour:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="spinBox_2" />
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="spinBox" />
       </item>
       <item row="1" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Internet Filter</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Filter File:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filter_file" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="filter_browse" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>1</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip" >
            <string>Browse</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset>../../data/icons/fileopen.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Addres:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filter_address" />
         </item>
         <item>
          <widget class="QPushButton" name="filter_add" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>1</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip" >
            <string>Add</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset>../../data/icons/edit_add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="filter_delete" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip" >
            <string>Delete</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset>../../data/icons/edit_remove.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="filter_update" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset>../../data/icons/reload.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" >
        <widget class="QTreeWidget" name="filter_treeWidget" >
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Filtered Addresses</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QCheckBox" name="filter_enable" >
         <property name="text" >
          <string>Enable Internet Filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5" >
      <attribute name="title" >
       <string>Cashiers</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item rowspan="3" row="0" column="0" >
        <widget class="QTreeWidget" name="cashiers_treeWidget" >
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Cashier List</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="cashiers_password" >
           <property name="echoMode" >
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="cashiers_username" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="textLabel3_2" >
           <property name="text" >
            <string>Real Name:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="textLabel1_3" >
           <property name="text" >
            <string>Username:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="textLabel2_2" >
           <property name="text" >
            <string>Password:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="cashiers_realName" />
         </item>
        </layout>
       </item>
       <item row="2" column="1" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cashiers_addButton" >
           <property name="text" >
            <string>Add</string>
           </property>
           <property name="icon" >
            <iconset>../../data/icons/edit_add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cashiers_updateButton" >
           <property name="text" >
            <string>Update</string>
           </property>
           <property name="icon" >
            <iconset>../../data/icons/reload.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cashiers_deleteButton" >
           <property name="text" >
            <string>Delete</string>
           </property>
           <property name="icon" >
            <iconset>../../data/icons/edit_remove.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cashiers_reportsButton" >
           <property name="text" >
            <string>Reports</string>
           </property>
           <property name="icon" >
            <iconset>../../data/icons/find.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter_add</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>292</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter_delete</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>328</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter_update</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>364</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter_treeWidget</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>199</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter_enable</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>199</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter_browse</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>363</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cashiers_treeWidget</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>81</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cashiers_addButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>183</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cashiers_deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>330</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cashiers_updateButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>253</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cashiers_reportsButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>408</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
